artview/components/plot_grid.py

```diff
@@ -254,11 +254,11 @@ def _fillLevelBox(self):
         self.levelBox.addItem("Level Window")
         # Loop through and create each level button
         if self.plot_type == "gridZ":
-            levels = self.Vgrid.value.axes['z_disp']['data']
+            levels = self.Vgrid.value.z['data']
         elif self.plot_type == "gridY":
-            levels = self.Vgrid.value.axes['y_disp']['data']
+            levels = self.Vgrid.value.y['data']
         elif self.plot_type == "gridX":
-            levels = self.Vgrid.value.axes['x_disp']['data']
+            levels = self.Vgrid.value.x['data']
 
```

Comment:
Wait now looking further below, either plot_grid or plot_grid_legacy is called within ARTView depending on version number, correct?